### PR TITLE
create guest

### PR DESCRIPTION
**Pull request by phucld95**

### CODE REVIEW

**Comment by dat-hedspi** [comment]
OK

File: restaurant_management/app/views/guests/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<%= form_for @guest, url:  guests_path, method: :post do |f| %>
```
Comment (yeunuthongminh) [inline restaurant_management/app/views/guests/_form.html.erb]: 1 cach

File: restaurant_management/app/views/guests/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <div class="col-md-4 col-sm-6">
+      <%= f.text_field :email, class: "form-control", 
+        placeholder: (t "guest.email") %>
+    </div>
+    <div class="col-md-4 col-sm-12">
+      <%= f.text_field :phone, class: "form-control", 
+        placeholder: (t "guest.phone") %>
+    </div>
+    <div class="col-sm-12">
+      <%= f.submit t "guest.next" %>
```
Comment (yeunuthongminh) [inline restaurant_management/app/views/guests/_form.html.erb]: thieu div dong

File: restaurant_management/app/views/guests/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<section id="contact" class="parallax-section">
+  <div class="container">
+    <div class="row  details-1">
+      <div class="col-md-offset-1 col-md-10 col-sm-12 text-center">
+        <h1 class="heading">About you</h1>
```
Comment (yeunuthongminh) [inline restaurant_management/app/views/guests/new.html.erb]: i18n

File: restaurant_management/app/models/guest.rb [hunk 4]
```diff
@@ -2,4 +2,20 @@ class Guest < ApplicationRecord
...
+  before_save :downcase_email
+
+  validates :name, presence: true, length: {maximum: 20}
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true, length: {maximum: 255},
+    format: {with: VALID_EMAIL_REGEX},
+    uniqueness: {case_sensitive: false}
+
+  validates :phone, presence: true, numericality: true,
+    :length => {:minimum => 8, :maximum => 11}
```
Comment (yeunuthongminh) [inline restaurant_management/app/models/guest.rb]: Convention

**Pull request merged**